.github/workflows/pack_publish.yml

```diff
@@ -407,9 +389,285 @@ jobs:
+            $indexer = Get-ChildItem -Path $path -Filter "indexer.exe" -Recurse -ErrorAction SilentlyContinue | Select-Object -First 1
+            
+            if ($searchd) {
+              echo "searchd.exe found at: $($searchd.FullName)"
+              # Get version
+              try {
+                & $searchd.FullName --version
+              } catch {
+                echo "Could not get version: $_"
+              }
+            } else {
+              echo "ERROR: searchd.exe not found!"
+              exit 1
+            }
+            
+            if ($indexer) {
+              echo "indexer.exe found at: $($indexer.FullName)"
+            }
+            
+            # Show directory structure
+            echo "Directory structure:"
+            Get-ChildItem -Path $path -Directory | ForEach-Object {
+              echo "  $($_.Name)"
+            }
+            
+            break
+          }
+        }
+        
+        if (-not $foundPath) {
+          echo "ERROR: Manticore installation directory not found"
+          exit 1
+        }
+      shell: pwsh
+    
+    - name: Start and Test Manticore
+      run: |
+        echo "=== STARTING MANTICORE SERVICE ==="
+        
+        # Find searchd.exe
+        $searchdPath = $null
+        $possiblePaths = @(
+          "C:\Program Files (x86)\Manticore\bin\searchd.exe",
+          "C:\Program Files\Manticore\bin\searchd.exe",
+          "C:\Manticore\bin\searchd.exe"
+        )
+        
+        foreach ($path in $possiblePaths) {
+          if (Test-Path $path) {
+            $searchdPath = $path
+            echo "Found searchd.exe at: $searchdPath"
+            break
+          }
+        }
+        
+        if ($searchdPath) {
+          # Create minimal config
+          $manticoreDir = Split-Path -Parent (Split-Path -Parent $searchdPath)
+          $configDir = Join-Path $manticoreDir "etc\manticoresearch"
+          $configPath = Join-Path $configDir "manticore.conf"
+          
+          if (-not (Test-Path $configDir)) {
+            New-Item -Path $configDir -ItemType Directory -Force | Out-Null
+          }
+          
+          # Create minimal working config
+          $configContent = @'
+        searchd {
+            listen = 127.0.0.1:9312
+            listen = 127.0.0.1:9306:mysql
+            listen = 127.0.0.1:9308:http
+            log = searchd.log
+            query_log = query.log
+            pid_file = searchd.pid
+            data_dir = data
+        }
+        '@
+          
+          Set-Content -Path $configPath -Value $configContent -Force
+          echo "Created minimal config at: $configPath"
+          
+          # Change to Manticore directory
+          Push-Location $manticoreDir
+          echo "Working directory: $(Get-Location)"
+          
+          # Create data directory
+          if (-not (Test-Path "data")) {
+            New-Item -Path "data" -ItemType Directory -Force | Out-Null
+          }
```

Comment:
Are you sure it's required. I can't see in the docs that the user should create data dir manually after installation. I also don't see that they should create a config. Don't we ship a config and an empty data dir along with the binaries? Won't it work?